.rubocop.yml

```diff
@@ -1,13 +1,28 @@
+################################################################################
+# RuboCop setup
+################################################################################
+require:
+  - rubocop-rspec
+
 AllCops:
+  NewCops: enable
   TargetRubyVersion: 2.6
 
-Style/StringLiterals:
-  Enabled: true
-  EnforcedStyle: double_quotes
-
-Style/StringLiteralsInInterpolation:
-  Enabled: true
-  EnforcedStyle: double_quotes
 
+################################################################################
+# Individual cop configurations
+################################################################################
 Layout/LineLength:
   Max: 120
+
+RSpec/MultipleExpectations:
+  Max: 5
+
+RSpec/MultipleMemoizedHelpers:
+  Max: 20
+
+# These 3 rules require a trailing comma, if and only if every item is on its
+# own line.
+Style/TrailingCommaInArguments:    { EnforcedStyleForMultiline: comma }
+Style/TrailingCommaInArrayLiteral: { EnforcedStyleForMultiline: comma }
+Style/TrailingCommaInHashLiteral:  { EnforcedStyleForMultiline: comma }
```

Comment:
Will this fix my complaint from the other PR? 🙏

Reply:
It sure will. It's a good idea, so I think we should drop this rule into all our rubocop confs.